doc/api/errors.md

```diff
@@ -563,6 +570,67 @@ found [here][online].
   encountered by [`http`][] or [`net`][] -- often a sign that a `socket.end()`
   was not properly called.
 
+<a id="nodejs-error-codes"></a>
+## Node.js Error Codes
+
+<a id="ERR_INVALID_ARG_TYPE"></a>
+### ERR_INVALID_ARG_TYPE
+
+The `'ERR_INVALID_ARG_TYPE'` error code is used generically to identify that
+an argument of the wrong type has been passed to a Node.js API.
+
+<a id="ERR_INVALID_CALLBACK"></a>
+### ERR_INVALID_CALLBACK
+
+The `'ERR_INVALID_CALLBACK'` error code is used generically to identify that
+a callback function is required and has not been provided to a Node.js API.
+
+<a id="ERR_REPL_INVALID_EVAL_CONFIG"></a>
+### ERR_REPL_INVALID_EVAL_CONFIG
+
+The `'ERR_REPL_INVALID_EVAL_CONFIG'` error code is used when both
```

Comment:
The error being thrown in the code is in fact `ERR_INVALID_REPL_EVAL_CONFIG`(`REPL` and `INVALID` are switched). 

Also I think `s/config/options` because that's what is written down in the docs, and those are in fact optional? `ERR_INVALID_REPL_EVAL_OPTION` with the current error message should be better.